libcodechecker/server/client_db_access_handler.py

```diff
@@ -675,8 +675,49 @@ def getRunResults_v2(self, run_ids, limit, offset, sort_types,
             session.close()
 
     @timeit
-    def getRunResultCount_v2(self, run_ids, report_filter, cmp_data):
+    def getRunReportCounts(self, run_ids, report_filter):
+        """
+          Count the results separately for multiple runs.
+          If an empty run id list is provided the report
+          counts will be calculated for all of the available runs.
+        """
+
+        results = []
+        session = self.__Session()
+        try:
+            if not run_ids:
+                res = session.query(Run.id).all()
+                run_ids = [r[0] for r in res]
+
+            filter_expression = process_report_filter_v2(report_filter)
+
+            count_expr = func.count(literal_column('*'))
+
+            q = session.query(func.max(Report.id), Run, count_expr) \
```

Comment:
What is the main purpose of this: `func.max(Report.id)`.
We query it but we don't use it (see: https://github.com/gyorb/codechecker/blob/d29313a58bbc8e721c5e5320cb7f3c8b1e9867ce/libcodechecker/server/client_db_access_handler.py#L705)

Same:
- https://github.com/Ericsson/codechecker/blob/version6/libcodechecker/server/client_db_access_handler.py#L1364
- https://github.com/Ericsson/codechecker/blob/version6/libcodechecker/server/client_db_access_handler.py#L1411

If we don't use it, remove it from the query.